line_obfuscator.go

```diff
@@ -92,16 +158,19 @@ func transformLineInfo(file *ast.File) ([]string, *ast.File) {
 		node := cursor.Node()
 		clearNodeComments(node)
 
+		if envGarbleTiny {
```

Comment:
I know this is because we clear all line data from the object files, but please also add a comment.